docker/docker-compose.dev.yml

```diff
@@ -41,6 +41,8 @@ services:
       - METADATA_SERVICE_AUTH_ENABLED=false
       - JAVA_TOOL_OPTIONS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5001
       - BOOTSTRAP_SYSTEM_UPDATE_WAIT_FOR_SYSTEM_UPDATE=false
+      - SEARCH_SERVICE_ENABLE_CACHE=false
```

Comment:
was search caching causing the failure in the test?

Reply:
This is not used when running smoke tests. This is run when developing locally (enables debugger for example 2 lines above), not used when running quickstart by default, nor in smoke-tests. I think that cache should be disabled for local development by default because one of the first things I did was to search, then realized I was missing data, loaded data, and the results were cached. This is a quality of life for local development.